web/impact/impact/v1/views/organization_detail_view.py

```diff
@@ -0,0 +1,63 @@
+# MIT License
+# Copyright (c) 2017 MassChallenge, Inc.
+
+from django.contrib.auth import get_user_model
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from impact.permissions import (
+    V1APIPermissions,
+)
+from impact.models import (
+    Organization,
+    Partner,
+    Startup,
+)
+
+INVALID_KEYS_ERROR = ("Received invalid key(s): {invalid_keys}. "
+                      "Valid keys are: {valid_keys}.")
+
+def organization_is_startup(organization):
+    return _organization_is(organization, Startup)
+
+def organization_is_partner(organization):
+    return _organization_is(organization, Partner)
```

Comment:
I think that `public_inquiry_email`, `organization_is_partner` and `organization_is_startup` should not live in `organization_detail_view.py` or in `organization_list_view.py`, since they are used in both, and non makes sense as the authority on that subject, at least no more than the other. 

I'd go for either `utils.py` or in `organization.py`, but I'm open for different suggestions.